website/docs/r/one_dashboard.html.markdown

```diff
@@ -357,7 +368,62 @@ Each widget type supports an additional set of arguments:
     * `initial_sorting` - (Optional) An attribute that describes the sorting mechanism for the table. This attribute requires specifying the following attributes in a nested block -
         * `name` - (Required) The name of column to be sorted. Examples of few valid values are `timestamp`, `appId`, `appName`, etc.
         * `direction` - (Required) Defines the sort order. Accepted values are `asc` for ascending or `desc` for descending.
+    * `data_format` - (Optional) A nested block that describes data format. See [Nested data_format blocks](#nested-data_format-blocks) below for details.
+        
+
+### Nested `data_format` blocks
+
+Nested `data_format` specifies the data structure format in the table. It defines how the data is organized and interpreted.This attribute requires specifying the following attributes in a nested block -
+
+  * `name` - (Required) This attribute mandates the specification of the column name to be formatted. It identifies which column the data format should be applied to.
+  * `type` - (Required) This attribute sets the format category for your data. Accepted values include `decimal` for numeric values, `date` for date/time values, `duration` for length of time, `recent-relative` for values referencing a relative point in time, `custom` for specific date/time formats and `humanized` to set autoformat as true.
```

Comment:
```suggestion
  * `type` - (Required) This attribute sets the format category for your data. Accepted values include - 
  - `decimal` for numeric values
  - `date` for date/time values
  - `duration`, for length of time
  - `recent-relative` for values referencing a relative point in time
  - `custom` to be used with date/time values, in order to select a specific format the date/time value would need to be rendered as
  - `humanized` to be used with numeric values, in order to enable Autoformat
```

I've added a few changes to each of these; also, I felt this (_maybe_) better as a bulleted list 🤔 (just a thought, let me know what you think). The list might also need to be indented to the right if needed